### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by corescan**
Updates for styling a scrollable form dialog, including a "hack" to expose the classname of the scrolling container.

### CODE REVIEW

File: src/Dialog.js [hunk 1]
```diff
@@ -37,7 +38,7 @@ export default class Dialog extends Component {
         {this.props.name ?
           <h1 className={css.title}>{this.props.name}</h1>
         : null}
-        <div className={css.body}>
+        <div className={classNames(css.body, getScrollingDialogClassname())}>
```
Comment (isuttell) [inline src/Dialog.js]: this doesn't change does it? Why do are using a function here?

Comment (corescan) [inline src/Dialog.js]: It does not change -- I exported this function so the CreateIssueDialog could access the class name. Doesn't need to be a function, just a const.

Comment (isuttell) [inline src/Dialog.js]: For simplicity lets just export the const

**Comment by isuttell** [comment]
Can you look at why the travis builds are failing too.

**Comment by ThinkTankShark** [comment]
Added to my list today to find out why travis is failing....

**Pull request merged**